app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,151 @@
+	]>
+
+<resources>
+	<string name="app_name">&appname;</string>
+	<string name="action_settings">Cài Đặt</string>
+	<string name="action_credits">Về tác giả </string>
+	<string name="action_change_log">Thay đổi truy cập</string>
+	<string name="action_picture_in_picture">Hình ảnh trong hình ảnh</string>
+
+	<string name="no_picture_in_picture">Thiết bị của bạn không hỗ trợ hình ảnh này trong hình ảnh.</string>
+
+	<!-- List -->
+	<string name="inaccessible">(không thể truy cập)</string>
+	<string name="go_home_folder">Không thể truy cập? Bấm vào đây để về trang chủ</string>
+	<string name="play_folder">Chạy trình chiếu ở đây</string>
+
+	<!-- Slideshow -->
+	<string name="title_activity_image">&appname;</string>
+
+	<string name="delete_button">Xoá</string>
+	<string name="delete_dialog_title">Xoá hình ảnh</string>
+	<string name="delete_dialog_message">Bạn có chắc không?</string>
+	<string name="image_deleted">Đã xoá hình ảnh</string>
+	<string name="image_not_deleted">Không thể xoá hình ảnh</string>
+
+	<string name="share_button">Chia sẻ</string>
+	<string name="share_via">Chia sẻ qua&#8230;</string>
+
+	<string name="image_detail_dimensions">Kích thước: %1$dx%2$d</string>
+	<string name="image_detail_size">Kích thước tập tin: %1$s</string>
+	<string name="image_detail_modified">Sửa đổi: %1$s</string>
+
+	<string name="toast_no_files">Không có tập tin nào để xem</string>
+
+	<string name="toast_paused">Tạm ngừng</string>
+	<string name="toast_resumed">Tiếp tục</string>
+
+	<string name="long_loading_warning">%1$s sẽ mất nhiều thời gian để tải.</string>
+	<string name="long_loading_skipping">%1$s đang mất nhiều thời gian để tải. Bỏ qua.</string>
+	<string name="long_loading_skip_action">Bỏ qua</string>
+
+	<!-- Image descriptions -->
+	<string name="image_description">Hình ảnh</string>
+	<string name="logo_description">Logo</string>
+
+	<!-- Credits -->
+	<string name="title_activity_credits">Về &appname;</string>
+	<string name="credits_creator">bởi <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">Slideshow là một ứng dụng đơn giản để xem hình ảnh trên điện thoại của bạn.
+	</string>
+	<string name="credits_content2">Không có tính năng đặc biệt, đây là ứng dụng đơn giản nhất trong các ứng dụng xem trình chiếu ở thị trường
+	</string>
+	<string name="credits_content3">Bạn có thể xem mã nguồn, truy cập lỗi hoặc yêu cầu các tính năng trên
+		<a href="https://github.com/Screaminghawk/android-slideshow">Github</a>.
+	</string>
+	<string name="credits_content4">Ứng dụng này hoàn toàn là phần mềm tự do nguồn mở; được cấp phép bởi <a href="https://tldrlegal.com/license/mit-license">MIT license</a>.
+	</string>
+	<string name="credits_content5">Cám ơn <a href="https://github.com/psyanite">Yan Tsui</a>
+		đã kiểm tra và thiết kế icon của &appname; .
+	</string>
+	<string name="credits_content6">Thông dịch của &appname; này được mang đến bởi
+		<a href="https://michael.standen.link">&author;</a>.
```

Comment:
You should replace this **your** name and website. It's a thank you that will show in the app for putting in the effort to make the translation :)